cmd/auth/main.go

```diff
@@ -1 +1,65 @@
-package auth
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"monorepa/pkg/auth"
+	"os"
+	"os/signal"
+)
+
+func defaultEnvConfig() {
+	// serever
+	os.Setenv("SERVER_HOST", "")
+	os.Setenv("SERVER_PORT", "8080")
+
+	// shutdown
+	os.Setenv("Server_Cancel_Timeout", "5")
+
+	// JWT config
+	os.Setenv("TOKEN_EXPIRE", "10") // minutes
+
+	// certificates
+	os.Setenv("CERT_VERSION", "1")
+	os.Setenv("CERT_PATH", "./pkg/storage/certificates")
+}
+
+func main() {
+	// init data
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt)
+	ctx := context.Background()
+
+	// add env part
+	addEnv := flag.Bool("envVars", false, "add default env variables")
+	flag.Parse()
+
+	if *addEnv {
+		defaultEnvConfig()
+	}
+
+	// init server and config
+
+	// server := auth.New(ctx)
+	server := auth.MockServerActions{}
```

Comment:
Mock server in executable?

Reply:
Temp part to pass all checks. Changed to normal exec.